.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,38 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.154.0/containers/rust
+{
+	"name": "Rust",
+	"build": {
+		"dockerfile": "Dockerfile"
```

Comment:
Can this be `image: mcr.microsoft.com/vscode/devcontainers/rust:0-1`? Seems like that would skip the first build.

Reply:
idk I took the defaults